vagrant/playbooks/pre_tasks/selinux-workarounds.yml

```diff
@@ -15,6 +15,15 @@
     - ansible_facts.os_family == 'RedHat'
     - ansible_facts.distribution_major_version | int != 7
 
+- name: Install loggers
+  yum:
+    name:
+      - rsyslog
+      - setroubleshoot
```

Comment:
So this purpose of this is to collect the errors as early as possible.

And the purpose of the other tasks to install these packages is to make the dev env consistent?

Also, why is setroubleshoot-server included in the other locations, but not here?

Reply:
actually, setroubleshoot-server is not needed, I forgot to remove it on the other places